digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -292,6 +292,13 @@ object ProjectDAO {
          """.execute
   }
 
+  def getProjectLinksFirstEly(projectId: Long): Seq[ProjectLink] = {
+     val query =
+      s"""$projectLinkQueryBase
```

Comment:
sql"""SELECT ELY FROM PROJECT_LINK WHERE PROJECT_ID=$projectId AND ELY IS NOT NULL AND ROWNUM < 2"""...headOption